### ISSUE TITLE
There is no XPlot.D3 nugget to use 

### ISSUE THREAD

**Issue by hiroCat** [fslaborg/XPlot#86]
First of all good job with the library! 

I tried to use the d3 to create some graphs but i didn't find in the public nuget the components.
I builded a nuget locally but it would be nice to have that to being able to distribute the code without having to insert all this repo.

**Comment by ErikSchierboom** [comment]
I have the same issue!

**Comment by cartermp** [comment]
I've updated tehe build to produce a package in #119, but there is only one chart type. Once this is merged, would you be interested in contributing some new chart types?
